dev/devicelab/lib/framework/runner.dart

```diff
@@ -88,7 +87,6 @@ Future<TaskResult> runTask(
   } finally {
     if (!runnerFinished)
       runner.kill(ProcessSignal.sigkill);
-    await cleanupSystem();
```

Comment:
Is there any way to check that the gradle daemon is not running?

Reply:
We could look for still running java processes in the before/after process check